### ISSUE TITLE
Unable to create a conda environment with zero package specs

### ISSUE THREAD

**Issue by stevemkim** [conda/conda#829]
Attempting to run `conda create` with zero package specs to install in the environment results in the error message:

```
Error: too few arguments, must supply command line package specs or --file
```

I report two problems:
1. Zero is a valid and useful number of packages to install. A conda environment that has zero packages installed is still a useful Python environment, because it is isolated from whatever cruft might be in the system-wide Python installation.
2. The error message is slightly misleading, because it implies that the error could be avoided by setting the `--file` option. However, if I provide an empty file, which the `specs_from_url` function happily parses, I get the same error message.

**Comment by isuruf** [comment]
> An environment without any packages is just an empty directory

With the latest changes, an environment without any packages still has files like `bin/conda`. Shall I open another issue for this

**Comment by kalefranz** [comment]
No, that is by design.  An empty environment in conda 4.3 will have the three conda executables/scripts `conda`, `activate`, and `deactivate`, along with an empty `conda-meta/history` file.

**Comment by isuruf** [comment]
@kalefranz, thanks for replying. My question is how do I make an empty environment? I can make an empty environment by installing a single package and then removing that package. Is there a way to make an empty environment directly?

**Comment by kalefranz** [comment]
Since conda 4.1 or 4.2, just `conda create -n my-new-env` or `conda create -p ~/my-conda-env`

**Comment by kalefranz** [comment]
Oh I see.  This issue was very old.  Yes, we implemented that in conda 4.2 I think.

**Comment by isuruf** [comment]
Thanks. it works in conda 4.2
